libmisc/salt.c

```diff
@@ -263,6 +267,78 @@ static /*@observer@*/const char *gensalt_bcrypt (void)
 }
 #endif /* USE_BCRYPT */
 
+#ifdef USE_YESCRYPT
+/* Default cost if not explicitly specified.  */
+#define Y_COST_DEFAULT 8
+/* Minimum cost.  */
+#define Y_COST_MIN 1
+/* Maximum cost.  */
+#define Y_COST_MAX 11
+/*
+ * Return a salt prefix specifying the cost for the YESCRYPT method.
+ */
+static /*@observer@*/const char *YESCRYPT_salt_cost (/*@null@*/long *prefered_cost)
+{
+	static char cost_prefix[5];
+        long cost;
+
+        if (NULL == prefered_cost) {
+	    cost = getdef_num ("YESCRYPT_COST_FACTOR", Y_COST_DEFAULT);
+        } else {
+            cost = *prefered_cost;
+        }
```

Comment:
Inconsistent indentation.

Reply:
Thank you! I couldn't see it on my editor since it renders the two on the same number of columns.